app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_4">4dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_16` appears to be unused
 Unused resources make applications larger and slow down builds.